test/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerServiceSpec.scala

```diff
@@ -52,12 +65,553 @@ class ClaimTransformerServiceSpec extends AnyWordSpec with Matchers with MockFac
   def mockGenerateIsoLocalDate(isoLocalDate: String): CallHandler0[String] =
     (mockDateGenerator.nextIsoLocalDate _: () => String).expects().returning(isoLocalDate)
 
-  val transformer = new DefaultClaimTransformerService(mockUuidGenerator, mockDateGenerator)
+  val transformer = new DefaultClaimTransformerService(mockUuidGenerator, mockDateGenerator, configuration)
 
   "Claim transformer" when {
 
     "passed a claim request" must {
 
+      "make an EIS submit claim request with basis of claim set to Incorrect Additional Code and enable-correct-additional-information-code-mapping flag is false" in {
+
+        val configuration: Configuration = Configuration(
```

Comment:
Can you move this into a method please?
You could pass in a boolean, and it would return the correct config.

Reply:
done